documents/read_acls_test.go

```diff
@@ -0,0 +1,125 @@
+// +build unit
+
+package documents
+
+import (
+	"testing"
+
+	"github.com/centrifuge/go-centrifuge/coredocument"
```

Comment:
since we moved the read acls to documents package, update the tests to use those functions

Reply:
I want to wait to update these tests because this whole package will be refactored anyway in a future PR which deals with the ReadACL methods and how they can be refactored into the CoreDocModel

Reply:
At the moment, there is Read ACL stuff in documents and coredocument, only one should the source. So lets either remove the read acl stuff from documents package and expose required methods in the core document.